test/common/common/utility_trie_speed_test.cc

```diff
@@ -48,13 +48,13 @@ template <class TableType> static void typedBmTrieLookups(benchmark::State& stat
   std::mt19937 prng(1); // PRNG with a fixed seed, for repeatability
   int num_keys = state.range(0);
   int key_length = state.range(1);
-  std::uniform_int_distribution<char> char_distribution('a', 'z');
+  std::uniform_int_distribution<> char_distribution('a', 'z');
```

Comment:
> The effect is undefined if this is not one of short, int, long, long long, unsigned short, unsigned int, unsigned long, or unsigned long long.

Does making the type unspecified actually fix this? If it's inferring a type and the arguments are chars won't it infer char type? Would it be better to make it explicitly short or int?

Reply:
This will be fixed since `<>` will be the default `int`, but yes it can be explictly specified, done.